www/docs/en/dev/config_ref/index.md

```diff
@@ -391,9 +391,8 @@ Examples:
 <preference name="AppendUserAgent" value="My Browser" />
 
 <!-- Windows only preferences -->
-<preference name="windows-phone-target-version" value="8.1" />
 <preference name="windows-target-version" value="8.1" />
-<preference name="Windows.Universal" value="10.0.10240.0" />
```

Comment:
From my understanding of https://github.com/apache/cordova-windows/blob/8cc627b8f5fb4a0959c4ef7a5994a4f358215fd6/template/cordova/lib/ConfigParser.js#L103-L170 this could never actually work and was probably a misunderstanding how the preference actually worked (what was written in this line is the fallback that is used if nothing is supplied here). Lines 398 and 399 show two working examples.